charts/app-config/values-integration.yaml

```diff
@@ -206,6 +206,147 @@ govukApplications:
           secretKeyRef:
             name: rails-secret-key-base
             key: SECRET_KEY_BASE
+- name: content-data-admin
+  helmValues:
+    workerEnabled: true
+    workerReplicaCount: 1
+    extraEnv:
+      - name: CONTENT_DATA_API_BEARER_TOKEN
+        valueFrom:
+          secretKeyRef:
+            name: signon-token-content-data-admin-content-data-api
+            key: bearer_token
+      - name: GOOGLE_TAG_MANAGER_ID
+        valueFrom:
+          secretKeyRef:
+            name: content-data-admin-google
```

Comment:
The naming of these seems a little odd / confusing to me. How about changing `google` to `google-tag-manager`, and removing the `manager-` prefix from the field names?